content/guides/migrating.md

```diff
@@ -618,35 +618,8 @@ Loaders are now cacheable by default. Loaders must opt-out if they are not cache
 
 ### Complex options
 
-webpack 1 only supports `JSON.stringify`-able options for loaders.
-webpack 2 now supports any JS object as loader options.
+__webpack v1__ only supports `JSON.stringify`-able options for loaders.
 
-Using complex options comes with one restriction. You may need to have a `ident` for the option object to make it referenceable by other loaders.
+__webpack v2__ now supports any JS object as loader options.
 
-Having an `ident` on the options object means to be able to reference this options object in inline loaders. Here is an example:
-
-`require("some-loader??by-ident!resource")`
-
-``` js
```

Comment:
Maybe move this example under the note, so folks know how it looked like :). Does \```diff work on the docs page ?

**webpack.config.js**
```diff
{
  test: /\.ext/
  use: { 
    loader: '...',
    options: {
-     ident: 'id',
      fn: () => require('./foo.js')
    }
  }
}
```